Document how to view a specific quota

AP-Hunt opened the pull request:
What
----

Our documentation showed how to view the big table of all quotas, but not how
to view the details of a specific quota.

How to review
-------------

Describe the steps required to test the changes.

1. Preview the content locally ([see README](https://github.com/alphagov/paas-tech-docs#preview)) and check that it renders as expected.
1. Manually check that any removed or renamed anchor tags are not in use ([linkchecker](https://github.com/linkcheck/linkchecker) doesn't do this).

Who can review
--------------
Anyone

jonathanglassman commented on source/documentation/managing_apps/quotas.md:
```diff
@@ -20,7 +20,31 @@ To see your organisation quota, run the command:
+
+``cf quota QUOTA``
+
+where QUOTA is the name of the quota your organisation has been assigned.
```
"...where `QUOTA` is the name of the quota that GOV.UK PaaS has assigned your org."

jonathanglassman commented on source/documentation/managing_apps/quotas.md:
```diff
@@ -20,7 +20,31 @@ To see your organisation quota, run the command:
 ``cf org YOURORG``
 
-where YOURORG is your organisation's name. (If you don't know the name, you can use ``cf orgs`` to find out).
+where YOURORG is your organisation's name. (If you don't know the name, you can use ``cf orgs`` to find out). This will produce an output which includes the name of the organisation's current quota:
```
organisation => org

AP-Hunt commented:
Fixed up the commit and addressed the two comments. Could you check I amended them the right way please, @jonathanglassman? I wasn't sure if I was meant to change organisation => org in every place in the sentences you highlighted.
